pages/careers/_open-positions.html

```diff
@@ -193,25 +198,60 @@ <h4>Your Ideal Background</h4>
+      </li>
+      <li>
+        <strong>Branding and Marketing Website.</strong> We have an improved understanding of our business that we'd like to reflect in a new public marketing website. We'll look to you lead the design effort here.
+      </li>
+      <li>
+        <strong>Design system.</strong> You'll be our first full-time designer so you will set the design direction for the entire company across all areas of design, working directly with the founders and closely with engineers.
+      </li>
+    </ul>
+
+    <p>Although you'll be the only professional designer on the team, our company cares deeply about design so you'll have many eager collaborators when it comes time to brainstorm.</p>
+
+    <h4>What skills are we looking for?</h4>
+
+    <ul>
+      <li>
+        <strong>Product sensibility.</strong> When we present our product vision, we'll look to you to offer opinions, insights, and methodologies to help us translate that vision into a fantastic product.
+      </li>
+      <li>
+        <strong>Design skill.</strong> At the end of the day, the designs you create should look great and be easy to use. Our hope is that you can make our users smile.
+      </li>
+      <li>
+        <strong>Domain expertise.</strong> We don't expect you to have a complete vision for our product on day 1, but we're looking for someone who either already understands our domain (DevOps, CI/CD, Infrastructure as Code, AWS), or who feels comfortable ramping up on it.
+      </li>
+      <li>
+        <strong>Bonus: HTML/CSS.</strong> If you can roll up your sleeves and review our engineers' work in HTML/CSS, you can help us make sure the final product is true to your vision.
+      </li>
+      <li>
+        <strong>Bonus: Engineering Background.</strong> If you happen to have an engineering background such as previous experience coding an app or a computer science degree, that may give you additional insight into how our product might especially delight users.
+      </li>
+    </ul>
+
+    <h4>What time zones do we work in?</h4>
+    <p>While Gruntwork <a href="#sane-working-hours">generally</a> hires anywhere between San Francisco and Berlin, for this role, you'll be working with a team based in the USA/Canada (GMT-7 to GMT-3). We prefer to hire candidates located in the USA or Canada, but are open to hiring in any country within those time zones. Unfortunately, this role is not available outside the GMT-7 to GMT-3 time zones.</p>
+
+    <a href="https://apply.workable.com/gruntwork-io/j/03345F97D6/apply/" class="btn btn-primary-hollow btn-sm d-inline-block btn-dpa"" role="button">Apply Now</a>
```

Comment:
Apply now button should be below each job role, and not just this one.

Reply:
Already addressed in #332.